data_architect_misc/data_transformer/qa_functions/common_comp_harm_qa_functions.py

```diff
@@ -767,7 +771,7 @@ def assert_GROSS_SPEND_column_values_have_two_decimals(
             df,
             [comp_harm_constants.GROSS_SPEND_COLUMN]
         )
-
+        import pdb; pdb.set_trace()
```

Comment:
Remove this. Always check for 'import pdb' and never leave them behind.

Reply:
Done.

Reply:
Done